app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java

```diff
@@ -58,6 +58,10 @@ public void onFinish() {
     }
 
     private static void updateTimeNotification(String machine, Context context, long timeLeft) {
+        if (notifcationIds == null || self == null) {
```

Comment:
Is there more info you can log to help figure out what is actually causing this?